### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nhatcuongti**
Add something to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# test-github
\ No newline at end of file
+# test-github
```
Comment (spartan-haobui) [inline README.md:1]: Why don't you eliminate this comment ?

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+Add something to readme
```
Comment (spartan-haobui) [inline README.md]: The message i want is "Add something to README"

**Review by spartan-haobui — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nhatcuongti) [reply README.md:1]: Oh sorry, i miss this

File: README.md (hunk 2 above)
Comment (nhatcuongti) [reply README.md]: Yeah, thanks again mr Hao. I miss this too

**Review by spartan-haobui — APPROVED** [review]
Ok let's go on

**Comment by spartan-haobui** [comment]
Finish feature A

**Pull request merged**